spec/index.html

```diff
@@ -1330,6 +1315,41 @@ <h2>Best Practices</h2>
           $type: WebContent
         -->
       </pre>
+
+      <div class="practice">
+        <p class="practicedesc">
+          <span id="bp-avoid-streams-in-html-scripts" class="practicelab">
+            Avoid stuffing more than one YAML document into a single `&lt;script>` HTML tag. Create one tag per document instead.
+          </span>
+        </p>
+      </div>
+
+      <p>Consider the following example.</p>
+
+      <pre class="example yaml"
+       data-transform="updateExample"
+       data-content-type="application/ld+yaml"
+       title="YAML-LD document stream embedded in HTML">
+      <!--
+      <script type="application/ld+yaml">
+        "@context": https://json-ld.org/contexts/person.jsonld
+        "@id": http://dbpedia.org/resource/John_Lennon
+        name: John Lennon
+        born: 1940-10-09
+        spouse: http://dbpedia.org/resource/Cynthia_Lennon
+      ---
+        "@context": https://json-ld.org/contexts/person.jsonld
+        "@id": http://dbpedia.org/resource/Cynthia_Lennon
+        born: 1939-09-10
+      </script>
+      -->
+      </pre>
+
+      <p>The YAML specification states in <a data-cite="YAML#912-document-markers">Document Markers</a> section that `---`, the document separator sequence, must be at the root level of the document, which means indentation level of 0.</p>
+
+      <p>Without the document separator, YAML content inside the `&lt;script>` block can be indented however the author of the document pleases; but the strict rule for indentation of the separator defies that apparent clarity.</p>
+
+      <p>It is probably best to avoid nesting YAML streams inside `&lt;script>` blocks; create one such block per document instead.</p>
```

Comment:
```suggestion
      <p>Aside from the document separator, YAML content inside the `&lt;script>` block can be indented however the author of the document pleases. The strict rule for indentation of the separator is an important exception.</p>

      <p>It is therefore best to avoid nesting YAML streams inside `&lt;script>` blocks; instead, create one such block per document.</p>
```